pkg/proto/info.go

```diff
@@ -63,14 +63,16 @@ func (i *FullAssetInfo) ToProtobuf(scheme Scheme) (*g.AssetInfoResponse, error)
 }
 
 type AssetInfo struct {
-	ID              crypto.Digest
-	Quantity        uint64
-	Decimals        byte
-	Issuer          WavesAddress
-	IssuerPublicKey crypto.PublicKey
-	Reissuable      bool
-	Scripted        bool
-	Sponsored       bool
+	ID                crypto.Digest
+	Quantity          uint64
+	Decimals          byte
+	Issuer            WavesAddress
+	IssuerPublicKey   crypto.PublicKey
+	Reissuable        bool
+	Scripted          bool
+	Sponsored         bool
+	IssueHeight       Height
+	IssueTxPosInBlock uint64 // TODO: size in bits can be reduced
```

Comment:
Really, why `uint64`? `uint32` would be enough as in scala implementation

Reply:
done